common/app/implicits/Collections.scala

```diff
@@ -3,24 +3,25 @@ package implicits
 import language.higherKinds
 import scala.collection.generic.CanBuildFrom
 import collection.mutable.{HashSet => MutableHashSet}
+import scala.collection.BuildFrom
 
 trait Collections {
 
   // thanks https://gist.github.com/1189097
   implicit class Seq2Distinct[T, C[T] <: Seq[T]](tees: C[T]) {
-    def distinctBy[S](hash: T => S)(implicit cbf: CanBuildFrom[C[T], T, C[T]]): C[T] = {
-      val builder = cbf()
-      val seen = MutableHashSet[S]()
-
-      for (t <- tees) {
-        if (!seen(hash(t))) {
-          builder += t
-          seen += hash(t)
-        }
-      }
-
-      builder.result
-    }
+//    def distinctBy[S](hash: T => S)(implicit cbf: CanBuildFrom[C[T], T, C[T]]): C[T] = {
```

Comment:
I didn't know how to fix this so I commented it out for now to get the next errors

Reply:
This is code where Frontend devs are extending the functionality of Scala's collection classes. Unfortunately there was a big improving refactor in Scala colllections going from Scala 2.12 to 2.13, and consequently the code would no longer be extended in the same way. See the migration guide here:

https://docs.scala-lang.org/overviews/core/collections-migration-213.html

In particular, the `implicit cbf: CanBuildFrom` stuff went away:

> Transformation methods no longer have an implicit `CanBuildFrom` parameter. This makes the library easier to understand (in source code, Scaladoc, and IDE code completion). It also makes compiling user code more efficient.

Extending Scala collections is now done in a different way - see the guide here : https://docs.scala-lang.org/overviews/core/custom-collection-operations.html - eg the part where a new `.sumBy()` method is added to existing collections. Another example of this is in https://github.com/rtyley/scala-collection-plus/blob/master/collection-plus/src/main/scala/com/madgag/scala/collection/decorators/package.scala !

What we want to do here, I'm not sure - it would depend on how often and where these methods are called, if we still want to implement them - maybe there are even better methods of providing this functionality?

Reply:
> maybe there are even better methods of providing this functionality?

LOL, yes there is! Turns out there is a brand new method in Scala 2.13 that is called the exact same name - [`distinctBy()`](https://www.scala-lang.org/api/2.13.x/scala/collection/Seq.html#distinctBy[B](f:A=%3EB):C) - and is called in exactly the same way. This means that all of `Seq2Distinct` can be deleted with Scala 2.13 ✨